packages/core/src/browser/quick-input/quick-command-frontend-contribution.ts

```diff
@@ -38,6 +38,11 @@ export class QuickCommandFrontendContribution implements CommandContribution, Ke
         commands.registerCommand(CLEAR_COMMAND_HISTORY, {
             execute: () => commands.clearCommandHistory(),
         });
+        commands.registerCommand({ id: 'workbench.action.closeQuickOpen' }, {
+            execute: () => {
+                this.quickInputService?.hide();
+            }
```

Comment:
minor: the brackets are unnecessary:

```suggestion
            execute: () => this.quickInputService?.hide()
```